core/src/toga/widgets/datepicker.py

```diff
@@ -15,7 +15,7 @@ class DatePicker(Widget):
             a new one will be created for the widget.
     """
 
-    MIN_WIDTH = 200
```

Comment:
This should probably be handled with a rehint based on the preferred size of the widget, rather than a hard coded value - but we can tackle that when we audit DatePicker.